src/lastFire.js

```diff
@@ -0,0 +1,24 @@
+const Entity = require('azure-entities');
```

Comment:
This looks good, but let's define this entity in `src/data.js` rather than in a new file (that's the more typical approach for Taskcluster services)